notebooks/lesson02_exploratory-data-analysis.ipynb

```diff
@@ -0,0 +1,2971 @@
+{
```

Comment:
<p>maybe a comment about duplicates could be useful. in the example you give you have a lot of duplicates in the left table and it might not be straightforward what happens if you then left join. no need to go through all combinations but raise awareness how it works.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>done</p>